# 基板温度(L-MESのみのみ)GET
---
float64 BoardTemp

